README.md

```diff
@@ -69,9 +69,45 @@ Usage
 You can call the *init* function if you want to configure JsonPollock - it is not mandatory, if you won't call it JsonPollock will be initialized with defaults.
 ```js
 JsonPollock.init({
-	maxAllowedElements: 50    // max DOM elements that will be rendere, dother elements will be ignored, default is 50.
+	maxAllowedElements: 100,
+	onAfterElementRendered: onAfterElementRenderedHandler
 });
 ```
+
+Supported options:
+
+|name | type | description | default |
+|---|---|---|---|
+| `maxAllowedElements` | Number |  max DOM elements that will be renderes, other elements will be ignored. | 50 |
+| `onAfterElementRendered` | Function | A callback function that will be invoked after each render on an element, it allows to customize the result DOM element. see the following for exact signature and usage example of the callback function. |  |
+
+Example callback function for `onAfterElementRendered`:
+```js
+/**
+* @param {HTMLElement} element html element that was rendered by json-pollock
+* @param {Object} template SC template of this element  
+* @returns {HTMLElement} manipulated html element
+*/
+(element, template) => {
+    // maniplate elements:
+    switch (template.type) {
+      case 'text':
```

Comment:
API users should not use bare strings; `template.type`s should be exported publicly as constants, e.g. `JsonPollock.TEMPLATE_TYPE.TEXT`.

do we have such thing?

Reply:
Nope, good point. I'll add it.

Reply:
done